src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Random randomNumber = new Random();
+        int numberForRandBall = randomNumber.nextInt(100);

```

Comment:
and use constant for 100
```suggestion
        Random random = new Random();
        int randomNumber = randomNumber.nextInt(100);

```

Reply:
в строке №8 может имелось ввиду int randomNumber = random.nextInt(100);
                                                    а не int randomNumber = randomNumber.nextInt(100);            ??
 потому что с randomNumber у меня выдает ошибку